src/flux/course.coffee

```diff
@@ -0,0 +1,35 @@
+_ = require 'underscore'
+
+{TaskActions, TaskStore} = require './task'
+{CrudConfig, makeSimpleStore, extendConfig} = require './helpers'
+
+CourseConfig =
+
+  loadPractice: (courseId) ->
+    @_local[courseId] ?= {}
+    @_local[courseId]['practices'] ?= []
+
+  loadedPractice: (obj, courseId) ->
+    @_local[courseId]['practices'].push(obj)
```

Comment:
okay, i'm a little confused about that.  i guess it sounded like to me every time you press "practice", the api pulls together a sampling of 5 exercise questions and so each time, it creates a new practice set.